pkg/nftlabs/nft.go

```diff
@@ -97,14 +117,21 @@ func (sdk *NftModule) MintBatch(meta []MintNftMetadata) ([]NftMetadata, error) {
 		return nil, err
 	}
 
-	_, err = sdk.getNewMintedBatch(receipt.Logs)
+	abiBatch, err := sdk.getNewMintedBatch(receipt.Logs)
 	if err != nil {
 		return nil, err
 	}
-
-	// TODO: process batch and fetch new nfts
-
-	return nil, nil
+	metas := []NftMetadata{}
```

Comment:
You could initialize with a fixed length because you know it already:

```go
	metas := make([]NftMetadata, len(meta))
```

minor speed/memory footprint improvement